miniapp/miniapp_triangular_multiplication.cpp

```diff
@@ -51,6 +51,7 @@ using dlaf::comm::CommunicatorGrid;
 using dlaf::common::Ordering;
 using dlaf::matrix::Matrix;
 using dlaf::matrix::MatrixMirror;
+using pika::this_thread::experimental::sync_wait;
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/eth-cscs/DLA-Future/pull/1112/commits/a49c151f29c8e52fe7b7c75859f2afdef2145943.